pyproject.toml

```diff
@@ -49,8 +49,6 @@ pytest11 = { "ucc-smartx" = "pytest_splunk_addon_ui_smartx.plugin" }
 [tool.poetry.group.dev.dependencies]
 pytest = ">=5.4, <7.3"
 Sphinx = "*"
```

Comment:
We can get rid of `Sphinx` dependency as well, right?

Reply:
Removed the `Sphinx` library, and updated ucc-gen library.